source/administration/important-upgrade-notes.rst

```diff
@@ -7,6 +7,9 @@ Important Upgrade Notes
 +----------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | If you’re upgrading from a version earlier than... | Then...                                                                                                                                                          |
 +====================================================+==================================================================================================================================================================+
+| v5.9.0                                             | Users with an outdated mobile version (such as a custom built app) might experience issues if the config.json is based on the new default.json                   |
```

Comment:
What's an outdated mobile version? If I'm not using a custom built app, can I have an outdated mobile app?

What action does the admin have to make? It says ```if the config.json is based on the new default.json ``DisableLegacyMFA`` ``` but I don't what to do about it.

Reply:
```suggestion
| v5.9.0                                             | If you are using the latest `default.json` that introduced the ``DisableLegacyMfa`` configuration setting defaulting to ``true``, users might face issues when they aren't running the latest mobile application (v1.17) together with multifactor-authentication. If the setting is not defined in the config file, the default value is `false` to avoid breaking current installations. If possible, ensure all users update to the latest mobile version and change the setting to ``true`` for additional security hardening. Otherwise change the setting to `false` if you are constrained to previous versions of the mobile application of Mattermost.            |
```
@jasonblais This should make it more clear, although the wording for sure can be enhanced. This is not a breaking change by default, I've discussed it in-depth with Katie how the roll out is planned. New installations will set this flag to true by default, old ones for now to false to avoid any breaking changes. For Gitlab I am not involved so far how the configuration is set, paging @hmhealey.

Reply:
As another note, if you are rolling out Mattermost via an Enterprise AppStore / MDM, you could be on a non up-to-date mobile version, as an example. Also if you are using a rooted phone, or build the Mattermost mobile app yourself.

This change does only affect MFA we handle internally, aka not any SSO providers. It also does not change the MFA functionality, but the way we check if a user has MFA enabled or not.

Reply:
GitLab configuration now works as it does for a regular server for most of the config settings, so there will be nothing special here for it.

Also, users may have an outdated mobile app if they have auto updates disabled or if they're on one of the versions of Android that we dropped support for